### ISSUE TITLE
Installation failure for latest commit (1f85d9c)

### ISSUE THREAD

**Issue by hszest** [MultiBUGS/R2MultiBUGS#2]
When running `devtools::install_github("MultiBUGS/R2MultiBUGS")` for the latest commit, my R (v3.5.1, with RStudio 1.1.456) complains:

```
Downloading GitHub repo MultiBUGS/R2MultiBUGS@master
from URL https://api.github.com/repos/MultiBUGS/R2MultiBUGS/zipball/master
Installing R2MultiBUGS
"C:/PROGRA~1/R/R-35~1.1/bin/x64/R" --no-site-file --no-environ --no-save --no-restore  \
  --quiet CMD INSTALL  \
  "C:/Users/USER/AppData/Local/Temp/RtmpWWnvPh/devtools140c586a376/MultiBUGS-R2MultiBUGS-1f85d9c"  \
  --library="C:/Users/USER/Documents/R/win-library/3.5" --install-tests 

* installing *source* package 'R2MultiBUGS' ...
** R
Error in parse(outFile) : 
  C:/Users/USER/AppData/Local/Temp/RtmpWWnvPh/devtools140c586a376/MultiBUGS-R2MultiBUGS-1f85d9c/R/bugs.script.R:193:25: unexpected ','
192:     if (!restart){
193:       "modelDistribute(",
                             ^
ERROR: unable to collate and parse R files for package 'R2MultiBUGS'
* removing 'C:/Users/USER/Documents/R/win-library/3.5/R2MultiBUGS'
In R CMD INSTALL
Installation failed: Command failed (1)
```

As previous commit can be installed successfully, maybe some recent modifications cause this problem.

Thanks

**Comment by hszest** [comment]
It seems that the change like
from
```
    if(!restart)"modelDistribute(", n.workers, ")\n",
    if(!restart && over.relax) 'over.relax("yes")\n',
```
to
```
    if (!restart){
      "modelDistribute(", n.workers, ")\n"
    },
    if (!restart && over.relax){
      "over.relax(\"yes\")\n"
    },
```
cause this problem, somehow the cat() function in R did not treat the curly braces well...

**Comment by rjbgoudie** [comment]
Thanks for the report. I think this is now fixed by 3497d261b146f46c6a7f33289ac67dace0feac3e - it just needed some c()s
